### DISCUSSION TITLE
Any way to adjust vertical and horizontal gaps separately?

### DISCUSSION THREAD

**Discussion by caniswim** [hyprwm/Hyprland#2567] [Q&A]
I use a 48-inch monitor and for some tasks it's ideal to be able to adjust the height and width of the windows.

On i3 and Sway I use these settings below to do this.

bindsym Mod1+Shift+Control+l gaps horizontal current plus 40

bindsym Mod1+Shift+Control+h gaps horizontal current minus 40

bindsym Mod1+Shift+Control+j gaps vertical current plus 40

bindsym Mod1+Shift+Control+k gaps vertical current minus 40

Is there something similar in Hyprland?

**Comment by MightyPlaza** [comment]
I don't think you can specify different vertical and horizontal gaps.
Apart from that you can change the gaps size with hyprctl
